datastore/db.go

```diff
@@ -2,16 +2,20 @@ package datastore
 
 import (
 	"os"
+
+	resty "github.com/go-resty/resty/v2"
 )
 
 // NewDB contructor with csv file
-func NewDB() (*os.File, error) {
+func NewDB() (*os.File, *resty.Client, error) {
 	//TODO Set file path with a Config
-	file, err := os.Open("datastore/pokemons.csv")
+	file, err := os.OpenFile("datastore/pokemons.csv", os.O_RDWR|os.O_CREATE, os.ModePerm)
+
+	client := resty.New()
```

Comment:
Move this client creation  after the err validation

Reply:
Done